sdk/synapse/Microsoft.Azure.Management.Synapse/src/Generated/Models/OperationResource.cs

```diff
@@ -39,7 +39,7 @@ public OperationResource()
         /// <param name="endTime">Operation start time</param>
         /// <param name="percentComplete">Completion percentage of the
         /// operation</param>
-        public OperationResource(string id = default(string), string name = default(string), string status = default(string), object properties = default(object), ErrorDetail error = default(ErrorDetail), System.DateTime? startTime = default(System.DateTime?), System.DateTime? endTime = default(System.DateTime?), double? percentComplete = default(double?))
+        public OperationResource(string id = default(string), string name = default(string), string status = default(string), object properties = default(object), ErrorResponse error = default(ErrorResponse), System.DateTime? startTime = default(System.DateTime?), System.DateTime? endTime = default(System.DateTime?), double? percentComplete = default(double?))
```

Comment:
this will result in a breaking change, please revert or have the breaking change team add the approval label to the spec pr